infrastructure/envs/dev/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.0"
+      version = "~> 6.0"
```

Comment:
Noting that this change doesn't affect the sandbox, only dev, test, prod configurations of the application.